src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static Random random = new Random();
+    private static Color[] colors = Color.values();
+
+    public static String getRandomColor() {
+        String randomColor = colors[random.nextInt(colors.length)].toString().toString();
+        return randomColor;
+    }
+}
```

Comment:
getDefaultColor() method should also be placed here as second method of ColorSupplier